legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -116,7 +116,7 @@ def validate_roles(incorporation_json) -> Error:
         err_path = '/filing/incorporationApplication/parties/roles'
         msg.append({'error': 'Must have a maximum of one completing party', 'path': err_path})
 
-    if incorporation_json['filing']['incorporationApplication']['nameRequest']['legalType'] == 'BC':
```

Comment:
FWIW, this field was called `requestTypeCd`.

Reply:
really? I haven't seen anything in the db with that key...is the front end passing that to the back end at any point?

Reply:
Nope. That's the name of the field in the NR. But I don't see anything wrong with getting the legal type from the business object. (It should have been set there when the IA was first created.)

In this screenshot, I notice a possibly new field, "entity_type_cd". I didn't see this in previous NRs we queried. (Also the type is CR -- corporation -- which is a current limitation of the namex that we query. But the business record should say it's a BC.)

![image](https://user-images.githubusercontent.com/10002889/83661149-94d80680-a57a-11ea-90f1-62884c542513.png)